storybook/stand-alone-documentation/welcome/contributors/contributors.jsx

```diff
@@ -0,0 +1,49 @@
+import { Contributor } from "../contributor/contributor";
+import { Orr, Hadas, Meirav, Sergey, Yael } from "../assets";
+import "./contributors.scss";
+
+const CONTRIBUTERS = [
```

Comment:
let's remove this file and content for now. It's not updated and I don't think it's critical. Vibe will have the latest one updated and once storybooks are connected, it will be redundant anyway